routes/shopifyAuth.js

```diff
@@ -83,6 +85,7 @@ module.exports = function createShopifyAuthRouter({
           }
 
           request.session.accessToken = accessToken;
+          request.session.shop = shop;
```

Comment:
this will error without having `express-session` hooked up, opened #21 about this